commands/gamble.js

```diff
@@ -0,0 +1,202 @@
+const _ = require('lodash');
+
+const gamble = {
+    // currently only allow one game globally
+    currentGame: null,
+
+    async process({ command, args, author, channel }) {
+        if (gamble[command]) {
+            if (gamble.currentGame && gamble.currentGame.channel.name !== channel.name) {
+                return currentGameError(channel);
+            }
+
+            if (!gamble.currentGame && command !== 'gamble') {
+                return;
+            }
+
+            await gamble[command](args, author, channel);
+        }
+    },
+
+    async gamble([pot], user, channel) {
```

Comment:
why is the first arg an array

Reply:
If you look at the upstream code, it parses text by turning `!command one two three` into:

```js
{
  command: 'command',
  args: ['one', 'two'],
}
```

Reply:
I figured just wanted to confirm